resources/views/auth/login.blade.php

```diff
@@ -69,19 +79,28 @@
             </div>
         </div>
 
-        <div class="col-6 col-lg-6 col-md-12 col-sm-12">
-            <div class="panel panel-default">
-                <div class="panel-heading h1">Register</div>
+        <div class="col-12 col-sm-12 col-md-12 col-lg-6">
```

Comment:
add `pt-3` to the class of this div, so that the register/login divs will clear eachother on mobile